File: Backend/TravelPlanner.Core/DataBaseModels/Hotel.cs
using System.Runtime.Serialization;

namespace TravelPlanner.Core.DataBaseModels
{
    [DataContract]
    public class Hotel
    {
        [DataMember]
        public string HotelId { get; set; }

        [DataMember]
        public string GeoId { get; set; }

        [DataMember]
        public string DestinationId { get; set; }

        [DataMember]
        public string LandmarkCityDestinationId { get; set; }

        [DataMember]
        public string Type { get; set; }

        [DataMember]
        public string Caption { get; set; }

        [DataMember]
        public string RedirectPage { get; set; }

        [DataMember]
        public float Latitude { get; set; }

        [DataMember]
        public float Longitude { get; set; }

        [DataMember]
        public string Name { get; set; }
    }
}

File: TravelPlannerApp/app/src/main/java/com/travelplanner/models/Hotel.kt
package com.travelplanner.models

class HotelWithDetails(val hotel: Hotel, val transport: List<HotelTransport>)

class Hotel(val hotelId: String, val geoId: String, val destinationId: String,
            val landmarkCityDestinationId: String, val type: String, val caption: String,
            val redirectPage: String, val latitude: Float, val Longitude: Float, val name: String)

class HotelTransport(val category: String, val transportLocations: List<TransportLocations>)

class TransportLocations(val name: String, val distance: String, val distanceInTime: String)